src/pages/kubecon-europe-2020.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import cx from 'classnames';
+import SEO from '../components/Seo';
+import { Button, Video } from '@newrelic/gatsby-theme-newrelic';
+import PageLayout from '../components/PageLayout';
+import ExternalLink from '../components/ExternalLink';
+import FeatherIcon from '../components/FeatherIcon';
+import kubeconHeader from '../images/kubecon-europe-2020/kubecon-europe-2020.jpg';
+import styles from './kubecon-europe-2020.module.scss';
+
+const KubeConPage = () => {
+  return (
+    <>
+      <SEO />
+      <PageLayout type={PageLayout.TYPE.SINGLE_COLUMN}>
+        <PageLayout.Header title="KubeCon and CloudNativeCon Europe 2020" />
+        <PageLayout.Content>
+          <section className={cx(styles.section, styles.twoColumn)}>
+            <div>
+              <h3>
+                New Relic welcomes you at <br />
+                Virtual Kubecon and CloudNativeCon Europe 2020!
+              </h3>
+              <p>
+                Learn more about the New Relic One platform, the only
+                observability platform that provides open, connected and
+                programmable observability for cloud-native environments. Join
+                Join us to dive into the New Relic One platform and our
+                Kubernetes cluster explorer.
+              </p>
+              <Button
+                as={ExternalLink}
+                variant={Button.VARIANT.PRIMARY}
+                href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/?utm_campaign=FY21-Q2-DEV-K8-EMEA-ELEV-OS-REG&utm_medium=OS&utm_source=ELEV&utm_content=REG&fiscal_year=FY21&quarter=Q2&gtm=DEV&program=K8&geo=EMEA"
+              >
+                Register here
+                <FeatherIcon
+                  className={styles.externalLinkIcon}
+                  name="external-link"
+                />
+              </Button>
+            </div>
+            <img
+              className={styles.img}
+              src={kubeconHeader}
+              alt="KubeCon and CloudNativeCon Europe 2020"
+            />
+          </section>
+          <section className={styles.section}>
+            <h2>Attend one of our lightning talks</h2>
+          </section>
+          <section className={cx(styles.section, styles.assetTable)}>
```

Comment:
When sequence matters, as it does here, grids are a poor choice. I suggest using a table or list instead. For example:

| What | Who | When |
| ----- | ----- | ------ |
| We handle Prometheus, you keep Grafana | Samuel Vandamme | Mon Aug 17 @ 14:35 CEST |
| How to use and customize Helm charts | Douglas Camata | Mon Aug 17 @ 16:25 CEST |
| ... | ... | ... |

Reply:
We don't need to be fancy, a simple Markdown table for now should suffice until we can componentize it properly.

Reply:
The page is created in react, I don't know how to include markdown there.
I tried with an html table but that looks horrible. Actually using a markdown table on an .mdx page also doesn't look good in my opinion.
I am sorry but this goes beyond my skills and I just want get this out so I am hoping someone can help?